src/components/MovieRow.jsx

```diff
@@ -1,26 +1,22 @@
-import React,{Component} from "react";
-import ReactDOM from "react-dom";
+import React from "react";
 
-class MovieRow extends Component{
+import "./MovieRow.css";
```

Comment:
importing CSS file, converting class component to functional component, the reason for that is because functional components can use react hooks which are way simpler and cleaner than the options we have when using class components.